packages/fluxible-router/docs/api/README.md

```diff
@@ -1,29 +1,21 @@
 # API Documentation
 
-`var Router = require('fluxible-router')` exports the following:
-
  * [`handleHistory`](handleHistory.md)
  * [`handleRoute`](handleRoute.md)
  * [`navigateAction`](navigateAction.md)
  * [`NavLink`](NavLink.md)
  * [`RouteStore`](RouteStore.md)
+ * [`HistoryWithHash`](HistoryWithHash.md)
 
-or if you're using ES6:
+ES6 imports:
 
 ```js
-import {handleHistory, handleRoute, navigateAction, NavLink, RouteStore} from 'fluxible-router';
+import {
+  handleHistory,
+  handleRoute,
+  navigateAction,
+  NavLink,
+  RouteStore,
+  HistoryWithHash
+} from 'fluxible-router';
 ```
-
-## Addons
```

Comment:
why was this removed?

Reply:
Since `HistoryWithHash` is part of the main package now, I kept the same doc structure as for the other functions/classes. So an entry was added on line 8 for it.

Reply:
By the way, since the package is now tree-shakeable, there is no need to keep the addons in a separated folder. Also, when `fluxible-router` was changed to use babel, the import for this package got broken:

- v1: `import { HistoryWithHash } from 'fluxible-router/addons'`
- v2: `import { HistoryWithHash } from 'fluxible-router/dist/addons'`
- This PR:  `import { HistoryWithHash } from 'fluxible-router'`